db/migrate/20200417153503_add_unconfirmed_email_to_spree_users.rb

```diff
@@ -1,4 +1,4 @@
-class AddUnconfirmedEmailToSpreeUsers < ActiveRecord::Migration[5.2]
+class AddUnconfirmedEmailToSpreeUsers < ActiveRecord::Migration[5.1]
```

Comment:
I think we should switch all the migrations to the helper provided by solidus_support [here](https://github.com/solidusio/solidus_support/blob/master/lib/solidus_support/migration.rb). It will handle the Rails version for us, not sure why this gem is not using it honestly.